pkgs/top-level/python-packages.nix

```diff
@@ -169,6 +169,26 @@ in {
     };
   };
 
+  antfs-cli = buildPythonApplication rec {
+    name = "antfs-cli-${version}";
+    version = "git-2017-02-11";
```

Comment:
Since the version field is not actually used anywhere in the expression you can inline it into the name field. Note, however, that this will not be interpreted as you might expect. Nix will parse name as `antfs-cli-git` and the version to `2017-02-11`. This is because it starts the version at the first hyphen followed by a digit:
```
nix-repl> builtins.parseDrvName "antfs-cli-git-2017-02-11"
{ name = "antfs-cli-git"; version = "2017-02-11"; }
```

Also as per the [naming guidelines](https://nixos.org/nixpkgs/manual/#sec-package-naming) it should be `name = "antfs-cli-unstable-2017-02-11";`.

This comment also applies for the openant package below.